contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributions
+
+Contributions are always welcome :smile ! you can contribute in many ways. 
+
+## Reporting Bugs
+
+Report Bugs at https://github.com/sforaidl/jeta/issues
+
+When reporting bugs please 
+- Include the version and some information about your local environment setup. 
+- Way to reproduce the bug
+
+## Fixing Bugs: 
+
+Issues tagged with `help wanted`,`good first issues` and `bug` are open for everyone.
+
+## Get Started
+
+Follow the steps to setup jeta for local development process
+
+1) Fork the Jeta repo on github
+
+2) clone your fork on local machine using following terminal command. 
+```bash
+git clone https://github.com/<insert your github username>/jeta.git
+```
+
+3) Create a conda virtual environment using `environment.yml` file. 
+
+```bash
+conda env create -f environment.yml
+conda activate jeta_env
+cd jeta/
+python setup.py develop
```

Comment:
`python setup.py develop` can mess up the environment. It's better to use `pip install -e .`
Overall, it would be more convenient this way (since then you don't have to worry about conda and pip conflicting.

```bash
conda create -n jeta
conda activate jeta
conda install pip
pip install -r requirements.txt
cd jeta/
pip install -e .
```